docs/proto/node.md

```
@@ -2226,53 +2203,148 @@
+ 
  
+ <a name="akash/escrow/types/v1/account.proto"></a>
+ <p align="right"><a href="#top">Top</a></p>
+
+ ## akash/escrow/types/v1/account.proto
+ 
+
+ 
+ <a name="akash.escrow.types.v1.Account"></a>
+
+ ### Account
```

markdownlint-cli2 (0.17.2)

Headings must start at the beginning of the line

(MD023, heading-start-left)